index.html

```diff
@@ -352,17 +352,64 @@ <h4>Script Type</h4>
             <p class="issue" data-number="33"></p>
             <p>The definitions of the types of documents and the corresponding <code>ttm:role</code> values are:
               <ul>
-                <li><dfn>Original Language Dialogue List</dfn>: When the <code>ttm:role</code> value is <code>x-DUBBING_ORIGINAL_DIALOGUE_LIST</code>, the document represents a literal transcription of the dialogue and on-screen text in their original spoken/written language(s).
-                <aside class="example">If a movie contains dialogue in English and Hebrew, the <a>Original Language Dialogue List</a> will contain some <a>Script Events</a> in English and some in Hebrew. None of the <a>Script Events</a> is translated.</aside>
+                <li>
+                  <dfn>Original Language Dialogue List</dfn>:
+                  When the <code>ttm:role</code> value is <code>x-DUBBING_ORIGINAL_DIALOGUE_LIST</code>,
+                  the document represents a literal transcription of the dialogue and on-screen text in their original spoken/written language(s).
+                <aside class="note"><a>Text</a> objects in the Original Language Dialogue List have their <a>Text Language Source</a> set to <a>Original</a>.</aside>
+                <aside class="example">If a movie contains dialogue in English and Hebrew,
+                  the <a>Original Language Dialogue List</a> will contain some <a>Script Events</a> in English and some in Hebrew,
+                  all marked as <a>Original</a> <a>Text Language Source</a>.
+                  None of the <a>Script Events</a> is marked as <a>Translation</a>.</aside>
```

Comment:
Similar comment. We should make sure that these notes are expressed elsewhere as normative statements.

Reply:
Done in 3cfe8db.